apps/websockets/consumers.py

```diff
@@ -0,0 +1,70 @@
+import json
```

Comment:
We might need to rename application. Websocket turned into notifications and now it turned into chat as well. So basically we can move chat to a separate app. So basically we will have a chat app, that is pretty simple and a notifications app which will have websocket implementation at this moment, but later on it might have something else

Reply:
Agreed